### PR TITLE
Added caching and compressing to the app

### PR DESCRIPTION
**Pull request by aditya1702**
Added Flask-Compress and Flask-Cache initializations.

### CODE REVIEW

**Comment by niranjan94** [comment]
I have added minification, caching and serving for static assets in PR #198 . Please verify.

File: open_event/__init__.py [hunk 1]
```diff
@@ -13,20 +13,29 @@
...
 
 def create_app():
     app = Flask(__name__)
+    compress = Compress()
+    cache=Cache(app,config={'CACHE_TYPE':'simple'})
     auto = Autodoc(app)
     cal = Calendar()
     event = Event()
+    
+    assets=Environment(app);
```
Comment (juslee) [inline open_event/__init__.py]: Codacy Issue found: [Unused variable 'assets'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406649190/issues/source?bid=3233864&fileBranchId=3233888#l31)

Comment (juslee) [inline open_event/__init__.py]: Codacy Issue found: [Unused variable 'assets'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406649190/issues/source?bid=3233864&fileBranchId=3233888#l31)

File: open_event/__init__.py [hunk 2]
```diff
@@ -13,20 +13,29 @@
...
 
 
 def create_app():
     app = Flask(__name__)
+    compress = Compress()
+    cache=Cache(app,config={'CACHE_TYPE':'simple'})
     auto = Autodoc(app)
     cal = Calendar()
     event = Event()
+
```
Comment (juslee) [inline open_event/__init__.py]: Codacy Issue found: [Trailing whitespace](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406649190/issues/source?bid=3233864&fileBranchId=3233888#l30)

Comment (juslee) [inline open_event/__init__.py]: Codacy Issue found: [Trailing whitespace](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406649190/issues/source?bid=3233864&fileBranchId=3233888#l30)

**Pull request closed without merging**